pallets/democracy/src/lib.rs

```diff
@@ -0,0 +1,1913 @@
+        NotDelegating,
+        /// The account currently has votes attached to it and the operation cannot succeed until
+        /// these are removed, either through `unvote` or `reap_vote`.
+        VotesExist,
+        /// The instant referendum origin is currently disallowed.
+        InstantNotAllowed,
+        /// Delegation to oneself makes no sense.
+        Nonsense,
+        /// Invalid upper bound.
+        WrongUpperBound,
+        /// Maximum number of votes reached.
+        MaxVotesReached,
+    }
+}
+
+decl_module! {
+    pub struct Module<T: Trait> for enum Call where origin: T::Origin {
+        type Error = Error<T>;
+
+        /// The minimum period of locking and the period between a proposal being approved and enacted.
+        ///
+        /// It should generally be a little more than the unstake period to ensure that
+        /// voting stakers have an opportunity to remove themselves from the system in the case where
+        /// they are on the losing side of a vote.
+        const EnactmentPeriod: T::BlockNumber = T::EnactmentPeriod::get();
+
+        /// How often (in blocks) new public referenda are launched.
+        const LaunchPeriod: T::BlockNumber = T::LaunchPeriod::get();
+
+        /// How often (in blocks) to check for new votes.
+        const VotingPeriod: T::BlockNumber = T::VotingPeriod::get();
+
+        /// The minimum amount to be used as a deposit for a public referendum proposal.
+        const MinimumDeposit: BalanceOf<T> = T::MinimumDeposit::get();
+
+        /// Minimum voting period allowed for an emergency referendum.
+        const FastTrackVotingPeriod: T::BlockNumber = T::FastTrackVotingPeriod::get();
+
+        /// Period in blocks where an external proposal may not be re-submitted after being vetoed.
+        const CooloffPeriod: T::BlockNumber = T::CooloffPeriod::get();
+
+        /// The amount of balance that must be deposited per byte of preimage stored.
+        const PreimageByteDeposit: BalanceOf<T> = T::PreimageByteDeposit::get();
+
+        /// The maximum number of votes for an account.
+        const MaxVotes: u32 = T::MaxVotes::get();
+
+        fn deposit_event() = default;
+
+        /// Propose a sensitive action to be taken.
+        ///
+        /// The dispatch origin of this call must be _Signed_ and the sender must
+        /// have funds to cover the deposit.
+        ///
+        /// - `proposal_hash`: The hash of the proposal preimage.
+        /// - `value`: The amount of deposit (must be at least `MinimumDeposit`).
+        ///
+        /// Emits `Proposed`.
+        ///
+        /// # <weight>
+        /// - Complexity: `O(1)`
+        /// - Db reads: `PublicPropCount`, `PublicProps`
+        /// - Db writes: `PublicPropCount`, `PublicProps`, `DepositOf`
+        /// # </weight>
+        #[weight = T::WeightInfo::propose()]
+        pub fn propose(origin, proposal_hash: T::Hash, #[compact] value: BalanceOf<T>) {
```

Comment:
My understanding is that these extrinsics are only meant to be callable from `pallets/simple_democracy`. If that is the case they need to be removed from this module definition.